lib/platform/dummy/index.js

```diff
@@ -0,0 +1,321 @@
+/* eslint-disable */
+const files = new Map();
+const prs = [];
+const branches = new Map();
+
+module.exports = {
```

Comment:
I'm sure there are better ways to do this, but as an experiment this worked very well.

Reply:
An alternative - maybe - would be that you use Jest to mock an existing platform such as GitHub. We do this in places for existing unit tests. Jest then automatically adds a mocked function for each existing export, each of them do nothing and return `undefined` when called. Then you can selectively add mocked functions where necessary and also use jest to count how many times the mock was called, what the parameters passed to it were, etc.

Reply:
For e2e/integration tests I usually find using a Fake easier to use and maintain than Mocks. Specially because we are testing from the edge of the public interface of the application, and the mocking usually happens way down the code, so it requires more knowledge on where and how to mock. 
Also, it's easier to reuse for other tests, For me, mocks makes more sense for ad-hoc implementation for a small test. But in this case, the mocks becomes very complicated because it touches different parts of the service, this way you don't care what are the actual things that you need. More than, if it worked or not.

This is a special use case, because usually I don't even need to do expects in the Fake object, in this case, I still need it, but it was way easier when I got a working version, not having to worry if I have it mocked in all the right places.